customer/src/main/java/no/unit/nva/customer/events/handler/CustomerTableDynamodbStreamToEventBridgeHandler.java

```diff
@@ -8,26 +8,39 @@
 import java.util.List;
 import no.unit.nva.commons.json.JsonUtils;
 import no.unit.nva.customer.events.aws.JacksonAttributeValueConverter;
+import no.unit.nva.customer.events.emitter.EventBridgeClientResourceUpdatedEventEmitter;
 import no.unit.nva.customer.events.model.ChannelClaim;
 import no.unit.nva.customer.events.model.ResourceUpdateEvent;
 import no.unit.nva.customer.events.producer.CustomerResourceUpdateEventsProducer;
 import no.unit.nva.customer.events.producer.DefaultCustomerResourceUpdateEventsProducer;
+import nva.commons.core.Environment;
 import nva.commons.core.JacocoGenerated;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
 
 public class CustomerTableDynamodbStreamToEventBridgeHandler
     implements RequestHandler<DynamodbEvent, List<ResourceUpdateEvent<ChannelClaim>>> {
 
     private static final Logger logger = LoggerFactory.getLogger(CustomerTableDynamodbStreamToEventBridgeHandler.class);
+    private static final String ENV_KEY_EVENT_BUS_NAME = "EVENT_BUS_NAME";
+    private static final String DETAIL_TYPE_RESOURCE_UPDATE_CHANNEL_CLAIM = "nva.resourceupdate.channelclaim";
```

Comment:
I think we have detail type with pattern like this:

PublicationService.Database.Update
PublicationService.DoiRequest.Update
ImportCandidates.DataEntry.Delete

What about detailType that matches pattern "Service.Type.Action", something like
`IdentityService.ChannelClaim.Update` or if more generic `IdentityService.Database.Update`